src/ReactTestWrapper.js

```diff
@@ -29,7 +29,8 @@ export default class ReactTestWrapper extends TestWrapper {
   }
 
   attr (name) {
-    return this.el.getAttribute(name) || undefined
+    const value = this.el.getAttribute(name)
+    return value !== null ? value : undefined
```

Comment:
Hmm...
Could you illustrate each case? I'm not sure to understand what are the difference between "empty attribute", "null attribute" and "undefined attribute", since `element.getAttribute()` will returns `null` if the attribute doesn't exist.

A `null` attribute or an `undefined` attribute is not possible, because the value can only be of type `string` afaik.